src/web_app/js/sdputils.js

```diff
@@ -226,7 +226,7 @@ function maybePreferVideoReceiveCodec(sdp, params) {
 // The format of |codec| is 'NAME/RATE', e.g. 'opus/48000'.
 function maybePreferCodec(sdp, type, dir, codec) {
   var str = type + ' ' + dir + ' codec';
-  if (codec === '') {
+  if (typeof codec === 'undefined' || codec === '') {
```

Comment:
I did not as I was thinking about the issue explicitly, your suggestion covers all the false cases. Will update.